dropwizard-jooq/src/main/java/com/bendb/dropwizard/jooq/jersey/DataSourceParam.java

```diff
@@ -0,0 +1,10 @@
+package com.bendb.dropwizard.jooq.jersey;
+
+import java.lang.annotation.*;
+
+@Documented
+@Retention(RetentionPolicy.RUNTIME)
+@Target({ElementType.PARAMETER})
+public @interface DataSourceParam {

```

Comment:
We don't need this annotation, we can just use the Named annotation. To do so we need to call named() inside of JooqBinder.configure().